File: Advanced/snakesandladders-cordapp/workflows/src/main/java/net/corda/samples/snl/service/GameSyncService.java
package net.corda.samples.snl.service;

import net.corda.core.node.AppServiceHub;
import net.corda.core.node.services.CordaService;
import net.corda.core.serialization.SingletonSerializeAsToken;

@CordaService
public class GameSyncService extends SingletonSerializeAsToken {

    private AppServiceHub serviceHub;

    public GameSyncService(AppServiceHub serviceHub) {
        this.serviceHub = serviceHub;
    }


}

File: kotlin-samples/Advanced/snakesandladders-cordapp/workflows/src/main/kotlin/net/corda/samples/snl/service/GameSyncService.kt
package net.corda.samples.snl.service

import net.corda.core.node.AppServiceHub
import net.corda.core.node.services.CordaService
import net.corda.core.serialization.SingletonSerializeAsToken


@CordaService
class GameSyncService(private val serviceHub: AppServiceHub) : SingletonSerializeAsToken()
